operations/helm/charts/grafana-agent/templates/serviceaccount.yaml

```diff
@@ -5,6 +5,9 @@ metadata:
   name: {{ include "grafana-agent.serviceAccountName" . }}
   labels:
     {{- include "grafana-agent.labels" . | nindent 4 }}
+    {{- with .Values.serviceAccount.additionalLabels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
```

Comment:
Can you add a corresponding test values.yaml file for this in `operations/helm/charts/grafana-agent/ci` and run `make generate-helm-tests` to generate the manifests for it? That way we can have regression tests in case this breaks.